frontend/src/modules/search/components/SearchBox.js

```diff
@@ -191,9 +219,18 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
             statuses: this.getInitialStatuses(),
             extras: this.getInitialExtras(),
             tags: this.getInitialTags(),
+            authors: this.getInitialAuthors(),
         });
     }
 
+    getAuthorsAndTimeRangeData = () => {
+        const { locale, project, resource } = this.props.parameters;
+
+        this.props.dispatch(
+            search.actions.getAuthorsAndTimeRangeData(locale, project, resource)
+        )
+    }
```

Comment:
I think you don't need this function, and you can call this action in the `componentDidMount` method of this class.

Reply:
See https://github.com/mozilla/pontoon/pull/1348#discussion_r312485133.